keys.go

```diff
@@ -118,6 +124,14 @@ func NewPublicKey(b [PublicKeySize]byte) (*PublicKey, error) {
 	}, nil
 }
 
+// NewKeypair creates a new keypair from a public key and secret key
+func NewKeypair(pk *PublicKey, sk *SecretKey) *Keypair {
```

Comment:
could you add a `*Keypair.PublicKey()` function to return the public key so that `PublicKey.Verify()` can be called? alternatively you can make a `Verify` function on `Keypair`

Reply:
I'm thinking ill make a Verify function on Keypair if that works for you!